node/sc/local_backend.go

```diff
@@ -188,7 +188,7 @@ func (lb *LocalBackend) SuggestGasPrice(ctx context.Context) (*big.Int, error) {
 	if err := checkCtx(ctx); err != nil {
 		return nil, err
 	}
-	return new(big.Int).SetUint64(lb.config.UnitPrice), nil
+	return new(big.Int).SetUint64(lb.config.Governance.KIP71.UpperBoundBaseFee), nil
```

Comment:
Don't we need to check KIP-71 hard fork is enabled or not?

Reply:
The tx pool did not have KIP71-enabled condition when I changed it. So, I just simply have put the `upperboundbasefee`. Now the latest code had such a guard and I updated this one under that policy (https://github.com/klaytn/klaytn/pull/1427/commits/9fc1d0df6861044fe9a2e65076a99f28ee9b2616). Thank you.